streams/src/test/java/org/apache/kafka/streams/processor/internals/ProcessorNodeTest.java

```diff
@@ -101,44 +88,16 @@ public void testMetrics() {
-        node.init(context);
 
-        final String[] latencyOperations = {"process", "punctuate", "create", "destroy"};
-        final String throughputOperation = "forward";
-        final String groupName = "stream-processor-node-metrics";
+        final String[] latencyOperations = {PROCESS};
         final Map<String, String> metricTags = new LinkedHashMap<>();
-        metricTags.put("processor-node-id", node.name());
-        metricTags.put("task-id", context.taskId().toString());
-        metricTags.put("client-id", "mock");
+        metricTags.put(PROCESSOR_NODE_ID_TAG, context.node.name);
+        metricTags.put(TASK_ID_TAG, context.taskId().toString());
 
         for (final String opName : latencyOperations) {
-            StreamsTestUtils.getMetricByNameFilterByTags(metrics.metrics(), opName + "-latency-avg", groupName, metricTags);
-            StreamsTestUtils.getMetricByNameFilterByTags(metrics.metrics(), opName + "-latency-max", groupName, metricTags);
-            StreamsTestUtils.getMetricByNameFilterByTags(metrics.metrics(), opName + "-rate", groupName, metricTags);
-            StreamsTestUtils.getMetricByNameFilterByTags(metrics.metrics(), opName + "-total", groupName, metricTags);
+            StreamsTestUtils.getMetricByNameFilterByTags(metrics.metrics(), opName + RATE_SUFFIX, STREAM_PROCESSOR_NODE_METRICS, metricTags);
+            StreamsTestUtils.getMetricByNameFilterByTags(metrics.metrics(), opName + TOTAL_SUFFIX, STREAM_PROCESSOR_NODE_METRICS, metricTags);
         }
-        assertNotNull(metrics.metrics().get(metrics.metricName(throughputOperation + "-rate", groupName,
-                                                               "The average number of occurrence of " + throughputOperation + " operation per second.",
-                                                               metricTags)));
-
-        // test "all"
-        metricTags.put("processor-node-id", "all");
-        for (final String opName : latencyOperations) {
-            StreamsTestUtils.getMetricByNameFilterByTags(metrics.metrics(), opName + "-latency-avg", groupName, metricTags);
-            StreamsTestUtils.getMetricByNameFilterByTags(metrics.metrics(), opName + "-latency-max", groupName, metricTags);
-            StreamsTestUtils.getMetricByNameFilterByTags(metrics.metrics(), opName + "-rate", groupName, metricTags);
-            StreamsTestUtils.getMetricByNameFilterByTags(metrics.metrics(), opName + "-total", groupName, metricTags);
-        }
-        assertNotNull(metrics.metrics().get(metrics.metricName(throughputOperation + "-rate",
-                                                               groupName,
-                                                               "The average number of occurrence of " + throughputOperation + " operation per second.",
-                                                               metricTags)));
-
-        final JmxReporter reporter = new JmxReporter("kafka.streams");
```

Comment:
I removed the part to test JmxReporter since it should be covered in `JmxReporterTest`, so removing them to cleanup some overlapped overage. Ditto elsewhere.